### DISCUSSION TITLE
某些api调用失败

### DISCUSSION THREAD

**Discussion by Lkhsss** [nonebot/discussions#188] [问答]
NoneBot2, 调用[公开api](https://github.com/botuniverse/onebot-11/blob/master/api/public.md#get_group_member_list-%E8%8E%B7%E5%8F%96%E7%BE%A4%E6%88%90%E5%91%98%E5%88%97%E8%A1%A8)时，显示`nonebot.adapters.onebot.exception.ActionFailed: <ActionFailed data=None, echo=1, msg=API_NOT_FOUND, retcode=404, status=failed, wording=API不存在>`

api如下：

1. set_restart
2. clean_cache

**Comment by yanyongyu** [comment]
api能不能用由你的协议端决定

**Reply by Lkhsss** [reply]
> 



> api能不能用由你的协议端决定

我是用的是OneBot V11 协议，而api是OneBot V11的官方文档提供的

**Reply by MingxuanGame** [reply]
go-cqhttp 并未支持 `set_restart` 和 `clean_cache`

**Reply by Lkhsss** [reply]
> go-cqhttp 并未支持 `set_restart` 和 `clean_cache`

那请问go-cqhttp有没有支持类似的api呢?

**Reply by MingxuanGame** [reply]
> 那请问go-cqhttp有没有支持类似的api呢?

暂无

**Reply by Lkhsss** [reply]
> 

好的，谢谢
